x-pack/plugins/siem/public/components/page/overview/overview_host_stats/index.tsx

```diff
@@ -6,75 +6,111 @@
 
 import {
   // @ts-ignore
-  EuiStat,
+  EuiDescriptionList,
```

Comment:
I would remove the `// @ts-ignore` above this as it's not needed and will end up causing your IDE to not report type checks when `EuiDescriptionList` has types.

Reply:
Will do!